includes/class-gdpr-audit-log.php

```diff
@@ -149,14 +149,7 @@ private static function email_mask( $email, $character = '-' ) {
 	 */
 	public static function export_log( $user_id, $token ) {
 		// see : https://github.com/trewknowledge/GDPR/blob/2.1.0/includes/class-gdpr-audit-log.php#L147.
-		if ( class_exists( 'Logger' ) ) {
-			//phpcs:disable WordPress.PHP.DevelopmentFunctions.error_log_var_export
-			Logger::log(
-				__CLASS__ . '::' . __FUNCTION__ . ' is not currently implemented.',
-				true
-			);
-			//phpcs:enable
-		}
+		throw new \DomainException('This function is not implemented.');
```

Comment:
Did not know `DomainException` was a thing, nice.

Reply:
I'm not 100% sure my usage is the way it's intended. I picked it up at either a user group, or a conference, and use it generally for things domain related loosely in the DDD sense. 

https://gnugat.github.io/2016/05/11/towards-cqrs-command-bus.html <-- something like that.